### PR TITLE
feat(activity-log): API for Activity Log

### PR DESCRIPTION
**Pull request by radulescuandrew**
https://wearetribus.atlassian.net/browse/TEO-307
https://wearetribus.atlassian.net/browse/TEO-308
https://wearetribus.atlassian.net/browse/TEO-309

@birloiflorian all without guard

### CODE REVIEW

File: backend/src/usecases/activity-log/create-activity-log-by-admin.usecase.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  ) {}
+
+  public async execute(
+    newLogRequestDto: CreateActivityLogByAdminDto,
+    admin: IAdminUserModel,
+  ): Promise<IActivityLogModel> {
+    // 1. Check if the volunteer exists in the organization and have a profile
+    const volunteer = await this.volunteerFacade.find({
+      id: newLogRequestDto.volunteerId,
+      organizationId: admin.organizationId,
```
Comment (birloiflorian) [inline backend/src/usecases/activity-log/create-activity-log-by-admin.usecase.ts:36]: Sa fie voluntar activ

File: backend/src/api/activity-log/presenters/activity-log-list-item.presenter.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    this.volunteer = log.volunteer
+      ? new IdAndNamePresenter({
+          id: log.volunteer.id,
+          name: log.volunteer.name,
+        })
+      : null;
+
+    this.activityType = log.activityType
+      ? {
+          ...new IdAndNamePresenter(log.activityType),
```
Comment (birloiflorian) [inline backend/src/api/activity-log/presenters/activity-log-list-item.presenter.ts:29]: Use the presenter with the icon

File: backend/src/api/activity-log/presenters/activity-log.presenter.ts [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+          name: log.volunteer?.user?.name,
+        })
+      : null;
+
+    this.createdByAdmin = log.createdByAdmin
+      ? new AdminUserPresenter(log.createdByAdmin)
+      : null;
+
+    this.activityType = log.activityType
+      ? new ActivityTypeListItemPresenter(log.activityType)
```
Comment (birloiflorian) [inline backend/src/api/activity-log/presenters/activity-log.presenter.ts:32]: Check if we need icon here

**Review by birloiflorian — CHANGES_REQUESTED** [review]
comment

File: backend/src/usecases/activity-log/create-activity-log-by-admin.usecase.ts (hunk 1 above)
Comment (radulescuandrew) [reply backend/src/usecases/activity-log/create-activity-log-by-admin.usecase.ts:36]: Done

File: backend/src/api/activity-log/presenters/activity-log.presenter.ts (hunk 3 above)
Comment (radulescuandrew) [reply backend/src/api/activity-log/presenters/activity-log.presenter.ts:32]: No. Done

**Review by birloiflorian — APPROVED** [review]

**Pull request merged**